tests/task_comments/test_dao_modify_comment.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from tests.task_comments.task_comments_base_setup import TaskCommentsBaseTestCase
+from app import messages
+from app.api.dao.task_comment import TaskCommentDAO
+
+
+class TestModifyCommentDao(TaskCommentsBaseTestCase):
+
+    def test_dao_modify_comment(self):
+        TaskCommentDAO.modify_comment(
+            user_id=1, _id=1, task_id=1, relation_id=2, comment="modified comment"
+        )
+
+        # Verify that task comment was modified
+        self.assertTrue(self.task_comment is not None)
+        self.assertTrue(self.task_comment.id is not None)
+        self.assertTrue(self.task_comment.task_id == 1)
+        self.assertIsNotNone(self.task_comment.creation_date)
+        self.assertIsNotNone(self.task_comment.creation_date)
+        self.assertEqual(self.task_comment.comment, "modified comment")
```

Comment:
There seems to be a scope for imporvement where you can also check if the `modified_date` is later than `creation_date`

https://github.com/anitab-org/mentorship-backend/blob/80114acd5937768f7680f9ac5542282127021c90/app/database/models/task_comment.py#L28

Reply:
Interesting point! I hadn't thought of that so probably with adding a new test the task_comments_dao would have to be split anyway, right? If my change is accepted this could be a new issue.